app/javascript/article-form/components/Help/EditorFormattingHelp.jsx

```diff
@@ -78,7 +78,7 @@ export const EditorFormattingHelp = ({ openModal }) => (
               <tr>
                 <td className="ff-monospace">&gt; quoted text</td>
                 <td>
-                  <span className="pl-2 border-0 border-solid border-l-4 border-base-50">
+                  <span className="pl-2 border-0 border-solid border-l-2 border-base-50">
```

Comment:
nitpick (non-blocking): Move this small change to a separate PR as it's not related to the CSS generator cleanup.

Reply:
it was the only element using that `border-l-4` class and since I was reducing other borders (`border-`, `border-r-`, `border-t-`, `border-b-` from unnecessary [sizes/widths](https://github.com/forem/forem/pull/15284/files#diff-73bcc9a8dbf65b2d301e5d0db91bb0e471cb1372f8a23dcd9c02a86418309b24L256-L295) (`...3`, `...4`, `...8`) I decided to sneak that one in as well. Otherwise I wouldn't do it. FWIW, this almost hasn't affected overall aesthetics - it's just thinner border but not significantly.